### PR TITLE
Wine_quality_dataset

### PR DESCRIPTION
**Pull request by NitiKaur**
@mlopatka 
 I have verified that everything is up to mark and made sure the Jupyter file is there.I am sorry I couldn't get to work earlier due to some personal reason and my PR had to be closed. I have made the changes as desired('recommend' feature dropped from the training dataset).Please review it(chanes made in the jupyter notebook) once.Thank You!

Objectives:-
1.To train and test classification model using Sklearn and compare them.
2.To preprocess the data, perform EDA and perform Hyperparameter tuning wherever necessary.

My work:-
I have used the 'Wine Quality' dataset for task 1.I have used SVM, KNN and Logistic Regression Algorithm for predicting if a particular wine brand is recommended or not.I achieved the objective of preprocessing the data, performing EDA and Hyperparameter tuning along with computing an evaluation metric and also plotting graphs for visualising the data. .Since the Recommend Feature had textual data, I converted it into Categorical data so as to ease the computability. I got the accuracy in the different models as follows-
SVM                 -  100% 
KNN                 -  86%
Logistic Regression -  100%
Since I got less accuracy with the KNN algorithm, I performed Hyperparameter-tuning to compute the best p-value and leaf size.
I have also visualised the data using the seaborn library plotting various features and then analysing the data henceforth. Since Quality had a linear recommend feature I dropped the feature in the linear SVM algorithm.

### CODE REVIEW

**Comment by mlopatka** [comment]
Thank you for the PR! Our apologies for the (long) delay in getting to review this contribution.
We took a pause from reviewing contributions made as part of the outreachy application process in order to prioritize planning the roadmap of the PRESC project moving forward. 

As a consequence of this delay, some outstanding PRs exhibit a degree of dissociation from the master branch that makes conflict resolution very complicated. We are capturing contributions that meaningfully contribute to the current direction of the project goals in an archival repo to help us systematically integrate the useful parts into the PRESC master branch. In some cases, this content will be merged and then migrated, and in some cases the specific content will be merged into the archival repo via a separate PR.

Once again, a sincere thanks for you interest and contribution to the PRESC project.
